packages/SwingSet/docs/virtual-objects.md

```diff
@@ -80,7 +86,74 @@ This defines a simple counter object with two properties in its state, a count a
   console.log(`${barCounter.getName()} count is ${barCounter.getCount()`); // "new bar count is 1"
 ```
 
-Note that the `init`, `actualize`, and `finish` functions are defined explicitly here for clarity of exposition, but in practice you'd usually declare them inline as arrow functions in the parameters of the `defineKind` call.
+Suppose you instead wanted to provide the the increment and decrement capabilities as independent facets.  A simplified version of the above (without the name property and `reset` method) might look like:
+
+```javascript
+  function initFacetedCounter() {
+    return {
+      counter: 0,
+    };
+  }
+  function actualizeFacetedCounter(state) {
+    function getCount() {
+      return state.counter;
+    }
+    return {
+      incrementer: {
+        inc() {
+          state.counter += 1;
+        },
+        getCount,
+      },
+      decrementer: {
+        dec() {
+          state.counter -= 1;
+        }
+      },
+    };
+  }
+  const makeFacetedCounter = defineKind('counter', initCounter, actualizeCounter);
+```
+
+Which you'd use like:
+
+```javascript
+  const { inc, dec } = makeFacetedCounter('foo');
+
+  incrementer.inc();
+  incrementer.inc();
+  console.log(`count is ${decrementer.getCount()`); // "count is 2"
+  decrementer.dec();
+  console.log(`count is ${incrementer.getCount()`); // "count is 1"
```

Comment:
One bit of ugliness with this example is confusion between facet names vs. method names.  I'm thinking that maybe the facet name should denote the purpose of the facet (so let's go with `incr` and `decr`, since you seem to like those), and then have a common method name for bumping the counter (say, `step`) that both use.